tracer/src/Datadog.Trace/Logging/DirectSubmission/DirectSubmissionLogLevelExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// <copyright file="DirectSubmissionLogLevelExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.Logging.DirectSubmission
+{
+    internal static class DirectSubmissionLogLevelExtensions
+    {
+        public const string Verbose = "Verbose";
+        public const string Debug = "Debug";
+        public const string Information = "Information";
+        public const string Warning = "Warning";
+        public const string Error = "Error";
+        public const string Fatal = "Fatal";
+
+        public static string GetName(this DirectSubmissionLogLevel logLevel)
+            => logLevel switch
+            {
+                DirectSubmissionLogLevel.Verbose => Verbose,
+                DirectSubmissionLogLevel.Debug => Debug,
+                DirectSubmissionLogLevel.Information => Information,
+                DirectSubmissionLogLevel.Warning => Warning,
+                DirectSubmissionLogLevel.Error => Error,
+                DirectSubmissionLogLevel.Fatal => Fatal,
+                _ => throw new InvalidOperationException("Unknown log level " + logLevel),
```

Comment:
[Nit] Should we add a Unit Tests to make sure we don't forget to fill this one out if we ever have the weird idea to add a new level?

Reply:
Added unit tests in https://github.com/DataDog/dd-trace-dotnet/pull/1945/commits/66aca70c6b5e11bdaed6f87433810fb9e03fb2c1

Reply:
Same comment as the other area, where I don't think it's safe the throw, and woould prefer "unknown" as discussed.